src/main/java/org/folio/circulation/resources/LostItemsRequiringActualCostResource.java

```diff
@@ -0,0 +1,205 @@
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+
+import org.folio.circulation.domain.Account;
+import org.folio.circulation.domain.FeeFine;
+import org.folio.circulation.domain.FeeFineOwner;
+import org.folio.circulation.domain.Item;
+import org.folio.circulation.domain.ItemLostRequiringActualCostsRepresentation;
+import org.folio.circulation.domain.LostItemRequiringCostsFee;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.LoanStatus;
+import org.folio.circulation.domain.MultipleRecords;
+import org.folio.circulation.infrastructure.storage.ServicePointRepository;
+import org.folio.circulation.infrastructure.storage.feesandfines.FeeFineOwnerRepository;
+import org.folio.circulation.infrastructure.storage.inventory.ItemRepository;
+import org.folio.circulation.infrastructure.storage.loans.LoanRepository;
+import org.folio.circulation.infrastructure.storage.loans.LostItemPolicyRepository;
+import org.folio.circulation.infrastructure.storage.users.UserRepository;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.RouteRegistration;
+import org.folio.circulation.support.http.client.CqlQuery;
+import org.folio.circulation.support.http.client.Offset;
+import org.folio.circulation.support.http.client.PageLimit;
+import org.folio.circulation.support.http.server.JsonHttpResponse;
+import org.folio.circulation.support.http.server.WebContext;
+import org.folio.circulation.support.results.Result;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+public class LostItemsRequiringActualCostResource extends Resource {
+  private static final String RECORD_KEY = "lostItems";
+  private static final String TOTAL_RECORDS_KEY = "totalRecords";
+
+  private final String rootPath;
+
+  public LostItemsRequiringActualCostResource(String rootPath, HttpClient client) {
+    super(client);
+    this.rootPath = rootPath;
+  }
+
+  @Override
+  public void register(Router router) {
+    RouteRegistration routeRegistration = new RouteRegistration(rootPath, router);
+    routeRegistration.getMany(this::getMany);
+  }
+
+  private void getMany(RoutingContext routingContext) {
+    final WebContext context = new WebContext(routingContext);
+    final Clients clients = Clients.create(context, client);
+    final Repositories repositories = new Repositories(clients);
+    final PageLimit pageLimit = PageLimit.limit(context.getIntegerParameter("limit", 100));
+    final Offset offset = Offset.offset(context.getIntegerParameter("offset", 0));
```

Comment:
Could you please include these params to the RAML definition. Following should work, I think:
```yml
traits:
    pageable: !include raml-util/traits/pageable.raml

/circulation:
  /reports/lost-items-requiring-actual-cost-fees:
    type:
      collection-get:
        exampleCollection: !include examples/lost-items-requiring-actual-cost-fees.json
        schemaCollection: lost-items-requiring-actual-cost-fees
    get:
      is: [pageable] 
```